inference-engine/src/vpu/graph_transformer/src/model/data.cpp

```diff
@@ -31,6 +31,10 @@ namespace vpu {
 // DataNode
 //
 
+bool DataNode::isUsed() const {
+    return numConsumers() > 0 || !childDataToShapeEdges().empty();
```

Comment:
Is output data considered unused? It is not clear for API users, I think.

Reply:
Actually it can be used in case it the output data is used as someone's shape. I know that `is used` attribute has a very broad meaning and may be treated as something else. Maybe `isConsumed` would be better?

Reply:
`isConsumed` is better for me, thanks

Reply:
Done.

Reply:
Still can be confusing with *casual* consumers. We need some term describing *to have at least on consumer or child data -> shape edge*